config/plugins.js

```diff
@@ -0,0 +1,35 @@
+const { resolve } = require('path');
+const ExtensionsPlugin = require('@redhat-cloud-services/frontend-components-config-utilities/extensions-plugin');
+
+module.exports = [
+  new ExtensionsPlugin(
+    {
+      /**
+       * These extensions are offered up as 'console-extensions.json' (a subset are currently supported in HAC-Core)
+       */
+      extensions: [
+        {
+          type: 'console.page/route',
+          properties: {
+            path: '/infra',
+            component: {
+              $codeRef: 'init',
+            },
+          },
+        },
+        {
+          type: 'console.navigation/href',
```

Comment:
FWIW, these extensions will become obsolete in the long run -- today this is what HAC-Core reads to build the connectivity.

Reply:
wondering if we should create an enum for these types and use it?

Reply:
This should be a stop-gap to make things work smoothly while it gets built out into `.json` files. The Core SDK does not consume like this and this is more so for testing / short term development. I suspect when things are good and going (ideally at Milestone 3) this will not be needed and we just use the official bundling mechanisms.